src/elements/content-sidebar/additional-tabs/MoreTabLoading.js

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import './AdditionalTabs.scss';
+import PlainButton from '../../../components/plain-button/PlainButton';
+import IconEllipsis from '../../../icons/general/IconEllipsis';
+
+const MoreTabLoading = () => {
+    return (
+        <PlainButton className="bcs-additional-tab-loading-btn" type="button" isDisabled>
```

Comment:
Can we avoid duplication of the outer wrapper?

Reply:
the outer wrapper being the plain button? If I switch the loading component to use a div since it's always disabled, I won't be able to abstract with the actual button.